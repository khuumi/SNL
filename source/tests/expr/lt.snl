58.2 < 91