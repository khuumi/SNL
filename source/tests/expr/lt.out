Binop (Float 58.2) Lt (Int 91)